genie_registry/clinical.py

```diff
@@ -823,6 +850,14 @@ def _validate(self, clinicaldf):
         )
         total_error.write(error)
 
+        # CHECK: YEAR DEATH against YEAR CONTACT
+        haveColumn = process_functions.checkColExist(
```

Comment:
Nit: snake_case over camelCase

Reply:
Nit: I usually prefer current tense. Like has_column

Reply:
I like this idea. I use this variable name to match what we have in the existing code. @rxu17 we can convert to snake_case when we start refactoring the code.

Reply:
I'm happy to tweak as much as I can within the scope of the original PR. @rxu17 let me know your thoughts

Reply:
Two comments:

- Let's use a more specific variable name and not reuse `haveColumn`, something like `has_contact_and_death_years` or related.
- Since this is a new variable you are creating that's not being used in the code elsewhere (other than your new code you're adding), feel free to use snake_case